### DISCUSSION TITLE
Use external emojis permissions bugged?

### DISCUSSION THREAD

**Discussion by philipaarseth** [discordjs/discord.js#7400] [Q&A] [answered]
Does anyone know if permissions for "Use external emojis" is bugged currently?

On my brand new server I invited my bot to it seems only the permission for everyone for "use external emojis" matter. 
No matter how much access I give my bot, before or after, if I turn off "use external emojis" for everyone, the bot will fail to post emojis and instead post `:emoji_name:`

Things I've tried
* giving my bot administrator
* creating a new role with "use external emojis" enabled
* setting permissions code 262144 for "use external emojis when inviting the bot to a server

A friends server already had it enabled and the only way he could get it to work is to turn it off and on again.

**Comment by BenjammingKirby** [comment, marked as answer]
is it an interaction reply? if so that's the reason
interaction replies are based on webhooks and webhooks follow the everyone role permissions for external emojis

**Reply by advaith1** [reply]
To expand on this, interactions are totally separate from bots (the application doesn't need to have a bot account at all, or it can have a bot account that isn't in the server, but it can still receive interactions). Therefore, interactions are not affected by bot permissions.

**Reply by philipaarseth** [reply]
Yes it is. 

Is there any way to get around that?

**Reply by imranbarbhuiya** [reply]
> Yes it is.
> 
> Is there any way to get around that?

No. I'd opened a discussion but didn't got any response yet.
https://github.com/discord/discord-api-docs/discussions/4423

**Reply by BenjammingKirby** [reply]
> Is there any way to get around that?

well, no. simply put this is just how it works currently, when it comes to this particular case, the bot permission doesn't matter, the everyone role needs to have the permission enabled for the interaction replies to be able to use external emojis

**Reply by philipaarseth** [reply]
Right, well let's hope discord devs get around to taking a look at that soon.
